pkg/playbook/app/vcertutil/vcertutil.go

```diff
@@ -241,3 +242,18 @@ func RefreshTPPTokens(config domain.Config) (string, string, error) {
 
 	return "", "", fmt.Errorf("no refresh token or certificate available to refresh access token")
 }
+
+func GeneratePassword() string {
```

Comment:
+1

Just adding the note that in `cmd/` code we have this:
`flags.keyPassword = fmt.Sprintf("t%d-%s.tem.pwd", time.Now().Unix(), randRunes(4))`
notice that it has `tem.pwd` instead of `temp.pwd`. Ideally we want this enhanced on the other places if applicable.

Reply:
This is a random password being generated, that users will never see or know about. The content of the string is irrelevant

Reply:
I didn't meant to let the user know about the password, rather I was suggesting we re-use this function in other places so it's more readable. Not for this PR scope though.